test/regression/issue/07740.test.ts

```diff
@@ -0,0 +1,29 @@
+import { bunEnv } from "harness";
+import { bunExe } from "harness";
+import { tempDirWithFiles } from "harness"
+
+it("duplicate dependencies should warn instead of error", () => {
+    const package_json = JSON.stringify({
+        devDependencies: {
+            vuex: "3.6.2"
```

Comment:
Let's use a simpler package to avoid increasing test times
```suggestion
            "empty-package-for-bun-test-runner": "1.0.0"
```